src/main/java/de/sldk/mc/config/PrometheusExporterConfig.java

```diff
@@ -21,6 +21,7 @@ public class PrometheusExporterConfig {
             metricConfig("jvm_memory", true, Memory::new),
             metricConfig("players_online_total", true, PlayersOnlineTotal::new),
             metricConfig("players_total", true, PlayersTotal::new),
+            metricConfig("whitelisted_players", true, WhitelistedPlayers::new),
```

Comment:
How would you feel about making it disabled by default?

```suggestion
            metricConfig("whitelisted_players", false, WhitelistedPlayers::new),
```

My gut feeling is that this is more of a 'niche' metric as not everyone uses whitelists. Am I wrong?